applications/launchpad/gui-react/src/splashscreen.ts

```diff
@@ -0,0 +1,31 @@
+let start: number | null = null
+let opacity = 1
+
+const fadeOut = (timestamp: number, splashscreenEl: HTMLElement) => {
+  if (!start) start = timestamp
```

Comment:
```suggestion
  if (!start) {
    start = timestamp
  }
```
({ } make it more readable)

Reply:
Thanks! I will merge as it is, and fix in another PR.